### DISCUSSION TITLE
new user:  installs fine on Mac OS, runs, but then disappears.

### DISCUSSION THREAD

**Discussion by vincentblacksdhadow** [apohl79/audiogridder#1272] [Help]
Hi,

This is a great idea for a product.

I am running mac 12.3.1, and I've installed the Audio Gridder Server app 1.2.0
It installed, ran a long query for all my plug ins, and then, well, the tile card fades away quickly and won't reopen.   I don't see audiogridder in any DAW, and I have to restart my computer to get it to run again, with, sadly, the same results.

I welcome any thoughts.

**Comment by apohl79** [comment]
do you see a tray icon?

**Reply by paulster44** [reply]
Mac Ventura M1

Same thing happens to me..
I allowed accessibility and screen control.
I don't see a tray icon
Both computers can ping each other
I See Audiogridder server running on activity monitor (2 instances, I'm guessing in and out)
Running plugin on PC side, sees server

**Comment by paulster44** [comment]
I got it working .. manually added ip in plugin.

**Reply by paulster44** [reply]
Still can't see server window to change settings though
